Makefile

```diff
@@ -63,6 +63,29 @@ endif
 .PHONY: all doc
 all: install
 
+.PHONY: dlv
+dlv:
+	go get github.com/derekparker/delve/cmd/dlv
```

Comment:
I would prefer to not install/target dev tool as it is not the purpose of this Makefile. That said I'm ok with debug target to add flags.

Reply:
@safchain - removed `dlv` target